src/app/courses-page/course/course.component.html

```diff
@@ -24,8 +24,27 @@ <h3 class="title">{{ course.title | uppercase }}</h3>
     <button class="edit-btn btn" (click)="onEdit()">
       <img src="assets/icons/ic_edit.svg" alt="img" class="icon" />Edit
     </button>
-    <button class="delete-btn btn" (click)="onDelete()">
+    <button class="delete-btn btn" cdkOverlayOrigin #trigger="cdkOverlayOrigin" (click)="isOpen = true">
       <img src="assets/icons/ic_delete.svg" alt="img" class="icon" />Delete
     </button>
   </div>
 </section>
+
+<ng-template
+  #overlayDir="cdkConnectedOverlay"
+  cdkConnectedOverlay
+  [cdkConnectedOverlayOrigin]="trigger"
+  [cdkConnectedOverlayOpen]="isOpen"
+  [cdkConnectedOverlayHasBackdrop]="true"
+  [cdkConnectedOverlayBackdropClass]="'dark-backdrop'"
```

Comment:
The binding here is unnesesary 
`[cdkConnectedOverlayBackdropClass]="'dark-backdrop'" -> cdkConnectedOverlayBackdropClass="dark-backdrop"`
`[cdkConnectedOverlayHasBackdrop]="true" -> cdkConnectedOverlayHasBackdrop`
`[cdkConnectedOverlayPanelClass]="'preview-dialog-panel'" -> cdkConnectedOverlayPanelClass="preview-dialog-panel"`

Reply:
changed
thanks)